codechecker_common/source_code_comment_handler.py

```diff
@@ -90,7 +90,7 @@ def __process_source_line_comment(self, source_line_comment):
         # Check for codechecker source code comment.
         comment_markers = '|'.join(self.source_code_comment_markers)
         pattern = r'^\s*(?P<status>' + comment_markers + r')' \
-                  + r'\s*\[\s*(?P<checkers>(.*))\s*\]\s*(?P<comment>.*)$'
+                  + r'\s*\[\s*(?P<checkers>([^\]]*))\s*\]\s*(?P<comment>.*)$'
```

Comment:
```suggestion
                  + r'\s*\[\s*(?P<checkers>[^\]]*)\s*\]\s*(?P<comment>.*)$'
```
Do we need the extra group there?